File: 0387-first-unique-character-in-a-string/0387-first-unique-character-in-a-string.java
class Solution {
    public int firstUniqChar(String s) {
        int l=s.length();
        int c=0;
        for(int i=0;i<l;i++)
        {
            for(int j=0;j<l;j++)
            {
                if(i!=j&&s.charAt(i)==s.charAt(j))
                {
                    c++;
                    break;
                }
                

            }
            if (c==0)
            return i;
             c=0;
        }
        return -1;
    }
}

File: 0387-first-unique-character-in-a-string/0387-first-unique-character-in-a-string.py
class Solution:
    def firstUniqChar(self, s: str) -> int:
        ch={}
        for c in s:
            ch[c]=ch.get(c,0)+1

        for i in range(len(s)):
            if(ch[s[i]]==1):
                return i

        return -1